src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -367,7 +367,11 @@ public Build scheduleBuild(Map<String, ?> params) {
             // so wait for it to be added and then disappear
             waitFor(waitFor(By.id("notification-bar"))).until(bar -> !bar.isDisplayed());
         } else {
-            clickLink("Build with Parameters");
+            if (Boolean.getBoolean("new-job-page.flag.defaultValue")) {
+                clickButton("Build with Parameters");
```

Comment:
this no longer uses a new page but a dialog - as such the result of this is asynchronous and as such this must be using something like `runThenHandleInputDialog(java.lang.Runnable,java.lang.String)`

I say something like as I do not beleive there is currently something for "do something that will cause a dialog, when the dialog appears to this, then click the default button").